### PR TITLE
upgrade guides_style_18f

### PR DESCRIPTION
**Pull request by afeld**
...and other gems, per [Federalist migration instructions](https://docs.google.com/document/d/1fY2jgL_u60JCVi90VDoizthif446QipkVYRrr4vdyG0/edit#heading=h.rbh747k7gmxc).

/cc @wslack @ccostino

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -120,8 +120,8 @@ repos:
 google_analytics_ua: UA-48605964-19
 
 back_link:
-  url: "https://pages.18f.gov/guides/"
-  text: Read more 18F Guides
+  url: https://guides.18f.gov/guides
```
Comment (ccostino) [inline _config.yml]: @wslack should this just be `https://guides.18f.gov` given what we were talking about earlier in `wg-documentation`?

File: go [hunk 2]
```diff
@@ -65,7 +65,9 @@ BASE_PROOFER_OPTS = {
     %r{https://github.com/18F/Accessibility_Reviews}i,
     %r{https://github.com/18F/DevOps}i,
     %r{https://github.com/18F/handbook}i,
-    %r{https://github.com/18F/writing-lab}
+    %r{https://github.com/18F/writing-lab},
+    # https://github.com/gjtorikian/html-proofer/issues/118
+    '#'
```
Comment (ccostino) [inline go]: 🤔 What is this for?

Comment (afeld) [inline go]: The "return to top" link.

https://travis-ci.org/18F/before-you-ship/builds/225712602#L236-L237

Comment (ccostino) [inline go]: Oh right, thanks!

File: _config.yml (hunk 1 above)
Comment (wslack) [inline _config.yml]: it will be, but not until sometime next week

Comment (wslack) [inline _config.yml]: I would just say pages.18f.gov/guides if you want to merge now; that will redirect correctly.

**Comment by afeld** [comment]
Ready to go!

**Pull request merged**